packages/framer-motion/src/motion/types.ts

```diff
@@ -265,6 +265,10 @@ export interface MotionAdvancedProps {
     inherit?: boolean
 }
 
+type ExternalMotionValues = {
+    [K in keyof MotionStyle]: MotionValue<number> | MotionValue<string>
```

Comment:
Can we be a bit smarter about the types here and exclude the non-MotionValue type from `MotionStyle[K]`?

Reply:
What do you mean by this? Regardless I also had to change this to `[key: string]` as more of the interesting use-cases involve values not included in `MotionStyle`

Reply:
Ah right, so then I guess it doesn't really matter anymore.